src/TextDateInput/TextDateInput.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TextInput } from '../TextInput'
+import { Box } from '../Box'
+import { Fieldset, FieldsetProps } from '../fields/Fieldset'
+
+export type FullDateFormat = {
```

Comment:
I actually renamed in in sign-up. We could match is so that it is `DateObjectType`